metpy/calc/tools.py

```diff
@@ -1126,3 +1135,47 @@ def _process_deriv_args(f, kwargs):
         raise ValueError('Must specify either "x" or "delta" for value positions.')
 
     return n, axis, delta
+
+
+def parse_angle(input_dir_str, deg2rad=False):
```

Comment:
Add a `@exporter.export` before the `def` here. That will take care of making sure `parse_angle` is accessible from `metpy.calc` directly.

Reply:
"Instead of the deg2rad kwarg, I think we should just be using MetPy's unit support. Thoughts @jrleeman ?" <- I think that's a really good idea to keep it consistent, but I'm not too sure where to start (other than it has to do something with pint).